dotnet-web/pipeline.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - label: ":package: Restore dependencies"
+    command: "dotnet restore"
+    plugins:
+      - docker#v5.9.0:
+          image: "mono:latest"
+
+  - wait
+
+  - label: ":white_check_mark: Run tests"
+    command: "dotnet test --no-restore --verbosity normal"
+    plugins:
+      - docker#v5.9.0:
+          image: "mono:latest"
+
+  - wait
+
+  - label: ":construction_worker: Build"
+    command: "dotnet build --no-restore"
+    plugins:
+      - docker#v5.9.0:
+          image: "mono:latest"
```

Comment:
Seeing a missing new line here makes me think we should run prettier over the pipeline yaml. I'll make a note!